### ISSUE TITLE
PROMPT_COMMAND='history -a' in ~/.bashrc not executed in bash after updating LP to 2.2.1 from 1.11

### ISSUE THREAD

**Issue by Johann-Tree** [liquidprompt/liquidprompt#855]
Bug report file: 
[bug-report.txt](https://github.com/user-attachments/files/18247084/bug-report.txt)

### Steps to Reproduce
1. I'm using Ubuntu 20.04.6 LTS with bash version 5.0.17(1)-release. Recently I updated liquidprompt from version 1.11 to recent stable version 2.2.1.
2.  I didn't change ~/.bashrc, have following lines in there: 
`PROMPT_COMMAND='history -a'`
and at the end 
`#[[ $- = *i* ]] && source ~/Downloads/programme/liquidprompt/liquidprompt`

### Expected Behavior
The file ~/.bash_history is still updated after each command.

### Current Behavior
Since updating liquidprompt, the bash history isn't updated directly anymore but only when exiting the shell.

**Comment by Rycieos** [comment]
Wow, that's quite a version jump.

Please double check that you are setting `PROMPT_COMMAND` before loading Liquid Prompt. See [the wiki entry](https://github.com/liquidprompt/liquidprompt/wiki/FAQ#prompt-timing-andor-error-display-isnt-working) for more information.

If you are sure that you are, check the value of `LP_OLD_PROMPT_COMMAND`:
```bash
printf '%q\n' "$LP_OLD_PROMPT_COMMAND"
```
and share that here. It should have `'history -a'` somewhere in it.

**Comment by Johann-Tree** [comment]
I double checked that the last line of my ~/.bashrc is
`#[[ $- = *i* ]] && source ~/Downloads/programme/liquidprompt/liquidprompt`
See attached my whole [.bashrc](https://github.com/user-attachments/files/18266869/default.bashrc.txt) file.

Value of `LP_OLD_PROMPT_COMMAND` is empty:

```shell
[:~]└2 $ printf '%q\n' "$LP_OLD_PROMPT_COMMAND"
''
```

**Comment by Rycieos** [comment]
Thanks for that. I confirmed it was a bug; I was able to reproduce it. It only happens on Bash older than 5.1. It was introduced in commit 0ce764653a5f000598b6ad11c974fcefb65832da, version `v2.1.0-beta.1`. The fix will probably end up in a v2.2.2 version soon.
